app/controllers/contacts_controller.rb

```diff
@@ -55,6 +55,7 @@ class ContactsController < ApplicationController
         new_profile_data: new_profile_data_key_to_value(contact_params[:newProfileData], profile_data_to_change),
         edit_profile_reason: contact_params[:editProfileReason],
         message: contact_params[:message],
+        document: attachments,
```

Comment:
You're passing in a plural "attachments" to a singular "document" field -- that is confusing me. Does `attachments` actually contain a list of multiple values?
If Yes, how can it be safely passed to `document` that seems to only contain one value?
If No, why is it called `attachments` instead of `attachment`?

Reply:
Sorry for the confusion, this only supports only one attachment for now. I have renamed it to `attachment` now.